tests/commands/injective/test_get_transaction.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+from unittest import IsolatedAsyncioTestCase
+from unittest.mock import AsyncMock
+from unittest.mock import MagicMock
+from unittest.mock import Mock
+
+from frontrunner_sdk import FrontrunnerIoC
+from frontrunner_sdk.commands.injective.get_transaction import GetTransactionOperation # NOQA
+from frontrunner_sdk.commands.injective.get_transaction import GetTransactionRequest # NOQA
+from frontrunner_sdk.commands.injective.get_transaction import OrderFailure
+from frontrunner_sdk.exceptions import FrontrunnerArgumentException # NOQA
+
+
+class TestGetTransactionOperation(IsolatedAsyncioTestCase):
+
+  def setUp(self) -> None:
+    self.deps = MagicMock(spec=FrontrunnerIoC)
+    self.flags = [59]
+    self.hashes = ["3LVlH0iM5ZvkrZd9yCOZP5F3bgSEjPm7oPEkkOl7ank="]
+
+  def test_validate(self):
+    req = GetTransactionRequest(transaction_hash="abc")
+    cmd = GetTransactionOperation(req)
+    cmd.validate(self.deps)
+
+  async def test_get_transaction_order_failures(self):
+    mock_attributes = [
+      Mock(key="flags", value=json.dumps(self.flags)),
+      Mock(key="hashes", value=json.dumps(self.hashes))
+    ]
+    mock_log = Mock(events=[Mock(type="injective.exchange.v1beta1.EventOrderFail", attributes=mock_attributes)])
```

Comment:
This can be found in `pyinjective.proto.injective.exchange.v1beta1.events_pb2`.

Reply:
Oh, I meant

```python
from pyinjective.proto.injective.exchange.v1beta1.events_pb2 import EventOrderFail
```

Reply:
```suggestion
    mock_log = Mock(events=[Mock(spec=EventOrderFail, attributes=mock_attributes)])
```

Reply:
It's probably fine as is.

Reply:
Unfortunately this doesn't work, maybe something about protobuf objects
```
Traceback (most recent call last):
  File "/Users/brianvanderschaaf/.pyenv/versions/3.9.9/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/Users/brianvanderschaaf/Documents/frontrunner/projects/frontrunner-sdk/tests/commands/injective/test_get_transaction.py", line 40, in test_get_transaction_order_failures
    result = await GetTransactionOperation(GetTransactionRequest(transaction_hash="abc")).execute(self.deps)
  File "/Users/brianvanderschaaf/Documents/frontrunner/projects/frontrunner-sdk/frontrunner_sdk/logging/log_operation.py", line 22, in wrapped
    value = await operation(instance, deps)
  File "/Users/brianvanderschaaf/Documents/frontrunner/projects/frontrunner-sdk/frontrunner_sdk/commands/injective/get_transaction.py", line 67, in execute
    order_failures = OrderFailure.from_injective_response(response)
  File "/Users/brianvanderschaaf/Documents/frontrunner/projects/frontrunner-sdk/frontrunner_sdk/commands/injective/get_transaction.py", line 32, in from_injective_response
    if event.type == clz.EVENT_ORDER_FAIL_TYPE:
  File "/Users/brianvanderschaaf/.pyenv/versions/3.9.9/lib/python3.9/unittest/mock.py", line 630, in __getattr__
    raise AttributeError("Mock object has no attribute %r" % name)
AttributeError: Mock object has no attribute 'type'
```